docs/src/main/sphinx/release/release-465.md

```diff
@@ -0,0 +1,32 @@
+# Release 465 (7 Nov 2024)
+
+## General
+
+* Add the {func}`cosine_similarity` function for dense vectors. ({issue}`23964`)
+* Add support for reading geometries in EWKB files with the `ST_GeomFromBinary`
+  function. ({issue}`23824`) 
+* Add support for the `ORDER BY` clause in aggregate functions used with a
+  window context. ({issue}`23929`)
+* Add Accumulator and Group by hash CPU time metrics to verbose explain plans. ({issue}`21925`)
```

Comment:
@martint @losipiuk @raunaqmorarka @sopel39 .. this replicates the wording in the explain plan a bit but I am not convinced this is right .. here or in the explain plan - see the PR

Reply:
should it be group-by-hash or "Group by hash" or something like that?

Reply:
I don't think we should document this in release notes. These are low level metrics not intended for end users.

Reply:
ok